src/fireblocks-sdk.ts

```diff
@@ -711,6 +725,20 @@ export class FireblocksSDK {
         return await this.apiClient.issuePostRequest(`/v1/vault/accounts/${vaultAccountId}/${assetId}`, {}, requestOptions);
     }
 
+    /**
+     * Creates a new asset within a list of existing vault accounts
```

Comment:
Did you consult with any technical writer in prior to this?
Please improve readability, remember this is a whole new functionality which users are supposed to understand from the gecko.
please elaborate more about each operation and params (on all endpoints).

Also, be consistent with uppercase/lower case in the docs